tests/core/test_i64.in

```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
```

Comment:
+1 for renaming all `.in` to either `.c` or `.cpp`, if you're interested in doing that. A number of times I've wanted to build one of those files manually from command line, which then requires an additional flag to emcc to specify the suffix.